src/legacy/core_plugins/data/public/timefilter/timefilter.ts

```diff
@@ -156,7 +156,7 @@ export class Timefilter {
     });
   };
 
-  createFilter = (indexPattern: IndexPattern, timeRange: TimeRange) => {
+  createFilter = (indexPattern: IndexPattern, timeRange?: TimeRange) => {
```

Comment:
// brain-pick

If you don't specify a `timeRange`, it will create a filter with the current time range?
Why did you have to add this?